cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -432,6 +432,11 @@ func (csr *ClusterStateRegistry) IsNodeGroupSafeToScaleUp(nodeGroup cloudprovide
 }
 
 func (csr *ClusterStateRegistry) getProvisionedAndTargetSizesForNodeGroup(nodeGroupName string) (provisioned, target int, ok bool) {
+	if len(csr.acceptableRanges) == 0 {
```

Comment:
Withe the change you will get N lines in logs saying `AcceptableRanges have not been populated yet. Skip checking` instead of N lines saying `Failed to find acceptable ranges for %v"` for different groups. 

It is hard for me to see a benefit here.

Reply:
Right, I agree it will repeat for N node groups. I think people just get confused if they see "Failed to find acceptable ranges for %v" in the logs and it indicates something is wrong. If we know we may call an uninitialized collection, I think it's better to add checks.

Reply:
I can vouch on being confused by seeing those messages and looking them up... :)

Reply:
Fine. Let's merge it the. I personally do not see that it helps. But it does not break anything neither.